docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -39,7 +39,7 @@ Download and install the latest GPU driver from the [official Intel download pag
   pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0
   ``` -->
 
-Download and install the [**Intel oneAPI Base Toolkit**](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html?operatingsystem=window&distributions=offline). During installation, you can continue with the default installation settings.
+Download and install the [**Intel oneAPI Base Toolkit 2024.0**](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html?operatingsystem=window&distributions=offline). During installation, you can continue with the default installation settings.
 
```

Comment:
This link no longer contains 2024.0 @hkvision @chtanch

Reply:
yes, we already realized that, recorded here: https://github.com/intel-analytics/ipex-llm/issues/10552
@Oscilloscope98 is helping on this.